config/vufind/config.ini

```diff
@@ -278,6 +278,7 @@ session_name = VUFIND_SESSION
 ; ILS driver that supports them; not all drivers support holds/renewals.
 [Catalog]
 driver          = Sample
+ils_prefix      = vtls
```

Comment:
If we are going to add a new setting here, we should add comments explaining its purpose... and maybe it needs a more specific name, like ils_id_prefix. But I wonder if we're better off passing it as a parameter to the custom functions instead of making it a config setting, since its usage is currently rather narrow, and it may cause confusion if we make it a high-visibility configuration setting.